oauth2client/contrib/django_util/storage.py

```diff
@@ -12,16 +12,78 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from oauth2client.contrib.dictionary_storage import DictionaryStorage
+"""Contains a storage module that stores credentials using the Django
+ORM."""
 
-_CREDENTIALS_KEY = 'google_oauth2_credentials'
+from oauth2client.client import Error, Storage
 
 
-def get_storage(request):
-    # TODO(issue 319): Make this pluggable with different storage providers
-    # https://github.com/google/oauth2client/issues/319
-    """ Gets a Credentials storage object for the Django OAuth2 Helper object
-    :param request: Reference to the current request object
-    :return: A OAuth2Client Storage implementation based on sessions
+class DjangoORMError(Error):
```

Comment:
Obligatory question: is there any standard-library or existing custom error class that makes sense to use here instead?